ipalib/util.py

```diff
@@ -170,6 +170,10 @@ def check_writable_file(filename):
     """
     if filename is None:
         raise errors.FileError(reason=_('Filename is empty'))
+
+    if os.path.isdir(filename):
+        raise errors.FileError(reason=_('Directory name found instead of '
+                                        'Filename'))
```

Comment:
It must be `%(fname)s` translation strings doesn't support format (in versions we need at least)